<header class="banner"><div class="banner">
  <div class="logos">
    <div><img title="Julia logo" src="../figures/julia-logo-color.svg" alt="Julia animated logo" width=35 /></div>
    <div>◌</div>
    <div><img title="Powered by MathJax" src="https://www.mathjax.org/badge/mj-logo.svg"
              alt="Powered by MathJax" width=65 />
    </div>
  </div>  <!-- logos -->
  <div class="cite_textbook cb7">4.21 <span class="authors"></span></div>
</div></header>





<div class="mcs-bs-callout mcs-bs-callout-info">

#### work time: 8 minutes

</div></div>


### Given   
 1-kg of water that is initially at 90°C with a quality of 10 percent occupies a spring-loaded piston–cylinder device. This device is now heated until the pressure rises to 800 kPa and the temperature is 250°C. 


### Find 
Determine the total work produced during this process, in kJ. 

### Get Organized

Every good home has a good diagram.

![figure, cb8, 4-21](../figures/ch04_cen98179_p04021_lg.jpg)

These things you should notice right away:
1. Closed system ⋮ constant mass.  
1. We can resolve both states ⋮ $(T_1,x_1),\ (P_2,T_2),$ all given.
1. Boundary work is the only form of work stated.
1. We have to account for a spring $\left(\Delta F_{\rm spring} = k \Delta x\right).$ 

We have two states we can resolve, but are not told the path the system takes to go from one to the other. All we need is a little critical thinking:

Our system pressure is dictated by contributions from the following:  
1. Atmospheric pressure $P_{\rm atm}$, which is constant;   
1. Piston weight $\left(\Delta P_{\rm piston} = m_{\rm piston}g/P_{\rm piston} \right),$ also constant; and   
1. Spring force $\left(\Delta F_{\rm spring} = k \Delta x\right)$, which is proportional to displacement. Considering constant area, it follows naturally that the spring's contribution to pressure is proportional to volume (ie, the relationship is linear: $\Delta P \propto \Delta V$.

### Attack

All of our work is boundary work.  We can resolve both states, and we know $\Delta P \propto \Delta V$. Hopefully, you realize this means our process forms a trapezoid on a PV diagram. [see this problem for deeper explanation](cb8.4-68.jl.ipynb)

![PV diagram](../figures/cb8,4-021,PVdiagram.png)

\begin{gather*}
 W_{\rm b,out} = \int^2_1 P\,{\rm d}V = \frac{1}{2} \left( P_1 + P_2 \right)\left( V_2 - V_1 \right) \\[6pt]
 P_1 = P_\text{sat@90°C} \quad
 P_2 = 800\,{\rm kPa} \quad
 V_1 = mv_\text{90°C, x=10%} \quad 
 V_2 = mv_\text{250°C,800kPa}
\end{gather*}


In [1]:
using Unitful

In [2]:
m  = 1u"kg"                    # given
P1 = 0.7014u"bar"              # table
P2 = 800u"kPa"                 # given
x1 = 0.1                       # given

vf1 = 1.0360e-3u"m^3/kg"       # table
vg1 = 2.361u"m^3/kg"           # table
v1 = vf1 + x1 * (vg1 - vf1)    
v2 = 0.2932u"m^3/kg"           # table

W_out = m * (P1 + P2) * (v2 - v1) / 2 |> u"kJ"   # trapezoid

println("W_out = ", round(typeof(W_out),W_out,digits=3))

W_out = 24.437 kJ


<div class="btn-success">&nbsp;<div class="badge"></div><i class="fa fa-bomb"></i><strong> Boom</strong></div>
<div class="panel-success"><div class="panel-heading"><div class="badge"></div> 

\begin{equation*}
   W_{out} = 24.437\,\ce{kJ}
\end{equation*}  


</div></div>

<footer style="background-color: #461d7c; color:rgba(253,208,35,0.35); line-height:2em; padding-left:1em; padding-right:0.5em; bottom:0; float:bottom; text-align:right;"><i>ψᵟ</i></footer>